posthog/migrations/0427_externaldataschema_sync_type.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 4.2.11 on 2024-06-11 17:59
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("posthog", "0426_externaldatasource_sync_frequency"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="externaldataschema",
+            name="sync_type",
+            field=models.CharField(
+                blank=True,
+                null=True,
```

Comment:
You could probably do what I did with sync_frequency and force merge this past the migration check if you want the default set. I don't think there are that many schemas where this would cause an issue.

Reply:
Does having `null=True` set the column to `null` and not the `default` value then?

Reply:
yeah